### PR TITLE
Add info on ssb-room to README

### PR DESCRIPTION
**Pull request by KyleMaas**
Adds information to the README for how to add WebSockets support to ssb-room.  See #145.

(Going to try to upstream this as well, but at least for now it would be publicly documented.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,30 @@ I made a [blog post][pub-setup] on how to run a server pub to relay messages to
 
 `npm run build` for developing and `npm run release` for a much smaller bundle.  You can also run `npm run inline` to genereate a single monolithic index.html file with all resources included.
 
+# Enabling WebSockets in ssb-room
+
+To run an ssb-room which this can connect to, you will need to enable WebSockets support.  This requires three things:
+
+1. Add a line to `connections` so configure the WebSockets port (external, key, and cert need to be customized for your installation):
```
Comment (arj03) [inline README.md:71]: so configure -> to configure. And this probably needs to a bit more information about where this connections can be found. Might be good to explain that the key and cert are only needed for secure web sockets.

**Comment by KyleMaas** [comment]
Upstreamed configuration here:

https://github.com/staltz/ssb-room/pull/27

File: README.md (hunk 1 above)
Comment (KyleMaas) [reply README.md]: How's that look?

**Comment by arj03** [comment]
Thanks

**Pull request merged**